src/test/java/com/google/api/codegen/testdata/library.proto

```diff
@@ -105,7 +105,7 @@ service LibraryService {
 
   // Adds comments to a book
   rpc AddComments(AddCommentsRequest) returns (google.protobuf.Empty) {
-    option (google.api.http) = { post: "/v1/{name=shelves/*}/comments" body: "*" };
+    option (google.api.http) = { post: "/v1/{name=bookShelves/*}/comments" body: "*" };
```

Comment:
What do you mean by "already made the change"? It hasn't been pushed to master - I'd personally prefer changes to master to be smaller than larger, given all else equal.